lib/mastalk/extensions.rb

```diff
@@ -24,6 +24,7 @@ def extensions
 
       Dir.foreach(SNIPPETS_FOLDER) do |file|
         next if file == '.' || file == '..'
+        next unless file.end_with?('.erb')
```

Comment:
I'd favour `Dir.glob` with a wildcard match for `*.erb` over `Dir.foreach` and this logic.